pyproject.toml

```diff
@@ -31,6 +31,8 @@ dependencies = [
     "jill >=0.9.2",
     "pyzmq >=21.0",
     "spine-items>=0.21.3",
+    # dagster versions lower that 1.5.7 do not support pendulum >= 3.0.0
+    "pendulum < 3.0.0",
```

Comment:
@PiispaH This limit should be in Spine-Engine's `pyproject.toml`. Can you please move it there?

Reply:
Okay, will do!